analysis.yml

```diff
@@ -21,12 +21,14 @@ study:
                     --repo=/dev/shm/spack-$(SPEC) \
                     --from-commit="" \
                     --to-commit="" \
-                    --spec=$(SPEC) \
+                    --spec=$(SPEC)
+            rm -rf /dev/shm/spack-$(SPEC)
           nodes: 1
           procs: 1
-          walltime: "00:02:00"
+          cores per task: 1
```

Comment:
I may be old fashioned but would expect `cores_per_task` here.  I know YAML allows spaces but I don't frequently see keys that have spaces in them.

Reply:
Ooops we already talked about this but I forgot to update it on here. Turns out this key comes from Maestro not from us so it's not under our control.

Reply:
yep we decided to do the maestro thing. when in Rome...